src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  PageBreadcrumb,
+} from '@patternfly/react-core';
+import {IBreadcrumb} from 'src/resources/BreadcrumbResource';
```

Comment:
I don't think we need to create a `resource` file for the breadcrum you can define the interface in this file. `resource` is for API related resources